bravado/client.py

```diff
@@ -215,6 +215,16 @@ def construct_params(self, request, op_kwargs):
             if not remaining_param.required and remaining_param.has_default():
                 marshal_param(remaining_param, None, request)
 
+    def log_warning_for_deprecated_op(self):
+        # TODO: Move this logic to bravado_decorators
+        if self.operation.op_spec.get('deprecated'):
```

Comment:
While functionally equivalent, specifying a default value of `False` here is a little more explicit and in line with the [spec](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#operation-object)
```python
if self.operation.op_spec.get('deprecated', False):
```